src/mlpack/tests/main_tests/pca_test.cpp

```diff
@@ -41,12 +40,10 @@ struct PCATestFixture
   }
 };
 
-BOOST_FIXTURE_TEST_SUITE(PCAMainTest, PCATestFixture);
-
 /**
  * Make sure that if we ask for a dataset in 3 dimensions back, we get it.
  */
-BOOST_AUTO_TEST_CASE(PCADimensionTest)
+TEST_CASE_METHOD(PCATestFixture, "PCADimensionTest", "[PCAMainTest]")
```

Comment:
For the main tests we introduced another tag - `BindingTests` see: 

https://github.com/mlpack/mlpack/blob/6ced8d236590c86249d66204d8c1991beeb61249/src/mlpack/tests/main_tests/kmeans_test.cpp#L52-L53

for an example, can you add the tag.